modules/000-common/images/kube-rbac-proxy/Dockerfile

```diff
@@ -1,11 +1,22 @@
 ARG BASE_ALPINE
-FROM flant/kube-rbac-proxy:v0.1.1@sha256:d8ce260d21190440b30fbe34c24824c8bef0f7afc9b4c3895823c11716e7a2ff as artifact
+ARG BASE_GOLANG_16_ALPINE
 
-FROM $BASE_ALPINE
-COPY --from=artifact /kube-rbac-proxy /kube-rbac-proxy
-
-EXPOSE 8080
-
-ADD entrypoint.sh /
+FROM $BASE_GOLANG_16_ALPINE as build
+WORKDIR /src/
+RUN apk add --no-cache git patch make
+RUN wget https://github.com/brancz/kube-rbac-proxy/archive/v0.11.0.tar.gz -O - | tar -xz --strip-components=1 -C /src
+COPY patches/001-stale-cache.patch /src
+COPY patches/002-config.patch /src
+COPY patches/003-livez.patch /src
+RUN patch -p1 < /src/001-stale-cache.patch && \
+    patch -p1 < /src/002-config.patch && \
+    patch -p1 < /src/003-livez.patch && \
+    make build && \
+    cp /src/_output/kube-rbac-proxy-linux-$(go env GOARCH) /kube-rbac-proxy
 
+FROM $BASE_ALPINE
+RUN apk add -U --no-cache ca-certificates && rm -rf /var/cache/apk/*
+COPY --from=build /kube-rbac-proxy /kube-rbac-proxy
+COPY entrypoint.sh /
 ENTRYPOINT ["/entrypoint.sh"]
+EXPOSE 8080
```

Comment:
I believe that Dockerfile is a more convenient and more long-living format rather than werf stapel. I would prefer Dockerfile if we are enough with its functionality.

Reply:
I think we must use unified format, but at this moment this is not a problem.